index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Design Your Own Framework</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="containter_12 spacer">
+        space
+      </div>
+      <div class="fixedNavbar container_12 header">
+        <ul>
+          <li><a href="">All</a></li>
+          <li><a href="">Images</a></li>
+          <li><a href="">Videos</a></li>
+          <li><a href="">Maps</a></li>
+          <li><a href="">|</a></li>
+          <li><a href="">MSN</a></li>
+          <li><a href="">Office</a></li>
+          <li><a href="">Outlook</a></li>
+        </ul>
+        <ul class="floatRight">
+          <li class="content2"><a href="">English</a></li>
+          <li class="content2"><a href="">Sign In</a></li>
+          <li>
+            <a href=""><img class="icons" src="./img/usericon.png" alt=""/></a>
+          </li>
+          <li>
+            <a href=""><img class="icons2" src="./img/menu.png" alt=""/></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="container_12 contentMargin">
+        <img class="icons3" src="./img/iconfinder_bing_294660.png" alt="" />
+        <input type="search" name="Search" class="search" />
+      </div>
+      <div class="container_8 spacer">
+        space
+      </div>
+      <div class="container_8 spacer">
+        space
+      </div>
+      <div class="container_8 spacer">
+        space
+      </div>
+      <div class="container_8 spacer">
+        space
+      </div>
+      <div class="container_4 aside">
+        <div class="floatRight">
+          <div class="customButton"><p><</p></div>
```

Comment:
You have a markup validation error in this line, you should delete the extra `<` in the `<p>` tag.

Reply:
It was a nasty error on my part should have used the code for "<" instead of the character, fixing it in the morning.